src/ripple/app/tx/impl/CreateCheck.cpp

```diff
@@ -209,13 +210,10 @@ CreateCheck::doApply()
     // destination's owner directory.
     if (dstAccountId != account_)
     {
-        auto const page = dirAdd(
-            view(),
+        auto const page = view().dirInsert(
             keylet::ownerDir(dstAccountId),
-            sleCheck->key(),
-            false,
-            describeOwnerDir(dstAccountId),
-            viewJ);
+            checkKeylet,
+            describeOwnerDir(dstAccountId));
```

Comment:
Nit: it may be worth replacing `sleCheck->key()` in the log calls below with the new local `checkKeylet.key`.